.github/workflows/ci.yml

```diff
@@ -36,12 +36,7 @@ jobs:
 
       - name: install ${{ matrix.toolchain }} toolchain
         id: install_toolchain
-        uses: actions-rs/toolchain@v1
```

Comment:
That repository is archived, and it doesn't seem like it's going to receive any updates anytime soon.

https://github.com/actions-rs/toolchain/pull/220